src/isValidCommitMesage.ts

```diff
@@ -13,6 +13,7 @@ const DEFAULT_COMMIT_TYPES = [
     "merge",
     "update",
     "wip",
+    "Update: "
```

Comment:
This doesn't seem to make sense to add. 

"Update README.md" doesn't contain the string "Update: "

```suggestion
```

Reply:
I see your proof though. How exactly does this work? https://github.com/ubiquity/action-conventional-commits/issues/1#issuecomment-1614455615

Reply:
Resolve the conversation based on the last comment from the pull request